### DISCUSSION TITLE
Include panel title after dashboard title in HTML title tag

### DISCUSSION THREAD

**Discussion by sbraz** [grafana/grafana#39926] [Dashboards, Panels & Visualizations]
Hello,
At the moment, when I view a dashboard, the HTML title tag includes the dashboard title, e.g. _Icinga2 - Grafana_. However, when I view a specific panel, the title does not change.

Would it please be possible to append the panel title to the `<title>` tag?

This would result in something like _Icinga2 - Load - Grafana _ which I could easily find in my browser history in case I want to quickly access this specific panel. This would also make it easier to identify multiple browser tabs open on different panels from the same dashboard.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!

**Reply by sbraz** [reply]
Issue created: #82493.
